File: Employee/PracticomClient/src/app/models/employeePost.model.ts
import { EmployeePositionPost } from "./employeePositionPost.model"

enum Gender {
    male = 1,
    female = 2
}
export class EmployeePost {
    firstName!: string
    lastName !: string
    tz!: string
    employmentStartDate!: Date
    dateOfBirth!: Date
    gender!: Gender
    status !: boolean
    positions!: EmployeePositionPost[]

}

File: Employee/PracticomAPI/Practicom/Models/EmployeePostModel.cs
using Practicom.Core.Entites;

namespace Practicom.API.Models
{
    public class EmployeePostModel
    {
       
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Tz { get; set; }
        public DateTime EmploymentStartDate { get; set; }
        public DateTime DateOfBirth { get; set; }
        public Gender Gender { get; set; }
        public bool Status { get; set; }
        public IEnumerable<EmployeePositionPostModel> positions { get; set; }



    }
}
